packages/styles/scss/components/content-item-horizontal-thumbnail/_content-item-horizontal-thumbnail.scss

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/imports';
+@import '../link-list/index';
+
+@mixin content-item-horizontal-thumbnail {
+  :host(#{$dds-prefix}-content-item-horizontal-thumbnail),
+  .#{$prefix}--content-item-horizontal__row {
+    @include carbon--make-row;
+
+    border-top: 1px solid $ui-03;
+
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    @include carbon--breakpoint(md) {
+      min-height: rem(320px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-09;
+    }
+
+    @include carbon--breakpoint(lg) {
+      min-height: rem(296px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--layout-05;
+    }
+
+    padding-left: $carbon--spacing-05;
+    padding-right: $carbon--spacing-05;
+  }
+
+  :host(#{$dds-prefix}-content-item-horizontal) ::slotted([slot='heading']),
+  :host(#{$dds-prefix}-content-item-horizontal-copy) ::slotted(:not([slot])),
```

Comment:
Haven't seen this `::slotted(:not([slot]))`, can you tell me what this selector is targeting?

Reply:
@annawen1 TBH that's a new one to me, too. I was once again copying what I was seeing already existing, e.g. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/0d0571e43b1732086681ac59f1f7193765b9d83b/packages/styles/scss/components/content-item-horizontal-media/_content-item-horizontal-media.scss#L30